endpoint/endpoint.go

```diff
@@ -169,6 +169,11 @@ func (e *Endpoint) WithSetIdentifier(setIdentifier string) *Endpoint {
 	return e
 }
 
+// EndpointsHaveEmptyTargets checks if an endpoint has no targets
+func EndpointsHaveEmptyTargets(ingEndpoints []*Endpoint) bool {
```

Comment:
nit: this doesn't appear to be specific to ingress endpoints in any way, so I recommend adjusting the variable name or moving this functionality out of the generic endpoints package to the ingress source

Reply:
Renamed in 5f8083ced80e40cf8df46a381d56d1c4b0f1639f